src/test/metaFilterRouteSpec.js

```diff
@@ -0,0 +1,193 @@
+// const express = require('express')
+var configUtil = require('../libs/sb-config-util')
+var _ = require('underscore')
+
+const metaFilterRoutes = [
+  'content/search',
+  'course/search',
+  'framework/category/search',
+  'framework/term/search',
+  'search',
+  'framework/list'
+]
+
+const nonFilterRoutes = [
+  'user/inbox',
+  'tenant/info/sunbird'
+]
+
+var filterMiddleware = require('../middlewares/filter.middleware')
+var httpMocks = require('node-mocks-http')
+
+var baseUrl = 'http://localhost:5000/v1/'
+var async = require('async')
+var contentMetaConfig = require('./contentMetaConfig')
+
+var allowedChannels = contentMetaConfig.allowedChannels
```

Comment:
same comment why again a variable do all the data manipulation in the Data file and use it directly...